0-complexity/openvcloud
# distribute ros on nodes

see https://github.com/0-complexity/openvcloud/issues/1572

tested on  be-g8-3

test scenario :
- list cpu nodes and sort them based on number of ROS they have.
- create new cloudspace.
- check the cpu node which the router os is created on, (from the VFW of the new cloudspace), it should be the one with the smallest number of the ros.
